plugins/kubernetes/test/models/kubernetes/build_job_executor_test.rb

```diff
@@ -0,0 +1,152 @@
+require_relative "../../test_helper"
+
+SingleCov.covered! uncovered: 3
```

Comment:
can you add a comment on why there are uncovered lines or possibly add some test to hit them ?